src/questions/usdt.yaml

```diff
@@ -0,0 +1,170 @@
+  hint: NoHint
+  explanation: Tether Limited issued its first stablecoin in 2014, originally named Realcoin.
+  answerKeys:
+    - A
+  difficultyLevel: Low
+  choices:
+    - content: In 2014
+      key: A
+    - content: In 2015
+      key: B
+    - content: In 2016
+      key: C
+    - content: In 2017
+      key: D
+- uuid: fe25f799-14d9-4381-b685-209b2a9cdfa4
+  type: SingleChoice
+  content: What is the stable asset that USDT is typically pegged to?
+  hint: NoHint
+  explanation: USDT is pegged to US dollar.
+  answerKeys:
+    - B
+  difficultyLevel: Low
+  choices:
+    - content: Gold
+      key: A
+    - content: US dollar
+      key: B
+    - content: Bitcoin
+      key: C
+    - content: Ethereum
+      key: D
+- uuid: 94139043-5530-46e2-98bb-24fc8b26794b
+  type: SingleChoice
+  content: In which year did Tether's Taiwanese banking partners end their relationship with the company?
+  hint: NoHint
+  explanation: Tether's Taiwanese banking partners end their relationship with the company in 2017, which jeopardized Tether's ability to facilitate the conversion of fiat currency to cryptocurrency and vice versa.
+  answerKeys:
+    - C
+  difficultyLevel: Low
+  choices:
+    - content: In 2015
+      key: A
+    - content: In 2016
+      key: B
+    - content: In 2017
+      key: C
+    - content: In 2018
+      key: D
+- uuid: 2c378743-d4a2-4f83-b3b2-899bf918f738
+  type: SingleChoice
+  content: Who is the current CEO of Tether Limited?
+  hint: NoHint
+  explanation: JL van der Velde is the current CEO of Tether Limited.
+  answerKeys:
+    - B
+  difficultyLevel: Low
+  choices:
+    - content: Reeve Collins
+      key: A
+    - content: JL van der Velde
+      key: B
+    - content: Craig Sellars
+      key: C
+    - content: Brock Pierce
+      key: D
+- uuid: 235c69d6-f26b-4872-9a43-1c67c6244bf7
+  type: SingleChoice
+  content: What is the main controversy surrounding Tether Limited and its affiliated exchange, Bitfinex?
+  hint: All the options seem correct.
+  explanation: Tether and Bitfinex have faced lot of criticism for price manipulation, lack of transparency and security issues.
+  answerKeys:
+    - D
+  difficultyLevel: Low
+  choices:
+    - content: The allegation of price manipulation
+      key: A
+    - content: The lack of transparency and audits
+      key: B
+    - content: Security and liquidity issues
+      key: C
+    - content: All of the above
+      key: D
+- uuid: 1c7c34d4-166d-4c24-b3c3-be8b4c430d8f
+  type: SingleChoice
+  content: How is USDT meant to be backed according to Tether Limited?
+  hint: NoHint
+  explanation: USDT is meant to be backed 1:1 by the stable asset, typically held in the form of US Treasury bills.
+  answerKeys:
+    - B
+  difficultyLevel: Low
+  choices:
+    - content: 1:1 by a basket of commodities such as gold and oil
+      key: A
+    - content: 1:1 by the stable asset, typically held in the form of US Treasury bills
```

Comment:
1:1 by the liquid stable asset, like assets in the form is USD case or in the form of US Treasury bills